pkg/tenant/api/v1/api.go

```diff
@@ -250,13 +250,15 @@ type TenantProjectPage struct {
 }
 
 type Project struct {
-	ID          string `json:"id" uri:"id"`
-	TenantID    string `json:"tenant_id"`
-	Name        string `json:"name"`
-	Description string `json:"description,omitempty"`
-	Status      string `json:"status"`
-	Created     string `json:"created,omitempty"`
-	Modified    string `json:"modified,omitempty"`
+	ID           string `json:"id" uri:"id"`
+	TenantID     string `json:"tenant_id"`
+	Name         string `json:"name"`
+	Description  string `json:"description,omitempty"`
+	OwnerName    string `json:"owner_name"`
+	OwnerPicture string `json:"owner_picture"`
```

Comment:
Can we return a `Member` object instead that is only partially updated with name and picture? 

```suggestion
	Owner    Member `json:"owner"`
```

Feels like this would be more maintainable.

Reply:
Yeah that makes sense, the `Member` object doesn't currently have a picture field but it probably makes sense to add it, maybe we want to return it on Member List as well?